bus/routes.go

```diff
@@ -181,6 +181,26 @@ func (b *Bus) consensusNetworkHandler(jc jape.Context) {
 	})
 }
 
+func (b *Bus) postSystemSQLite3BackupHandler(jc jape.Context) {
+	var req api.BackupRequest
+	if jc.Decode(&req) != nil {
+		return
+	}
+	switch req.Database {
+	case "main", "metrics":
+	default:
+		jc.Error(api.ErrInvalidDatabase, http.StatusBadRequest)
```

Comment:
I don't like it when an API returns a 400 without telling me what values are considered valid (when it comes to enums of course, not all the time).